bugbug/models/invalid_compatibility_report.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+
+import xgboost
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import Pipeline
+
+from bugbug import feature_cleanup, issue_features, utils
+from bugbug.model import IssueModel
+
+logger = logging.getLogger(__name__)
+
+
+class InvalidCompatibilityReportModel(IssueModel):
+    def __init__(self, lemmatization=False):
+        super().__init__(
+            owner="webcompat", repo="web-bugs", lemmatization=lemmatization
+        )
+
+        self.calculate_importance = False
+
+        feature_extractors = []
+
+        cleanup_functions = [feature_cleanup.CleanCompatibilityReportDescription()]
+
+        self.extraction_pipeline = Pipeline(
+            [
+                (
+                    "report_extractor",
+                    issue_features.IssueExtractor(
+                        feature_extractors, cleanup_functions, rollback=False
+                    ),
+                ),
+                (
+                    "union",
+                    ColumnTransformer(
+                        [
+                            (
+                                "first_comment",
+                                self.text_vectorizer(min_df=0.0001),
+                                "first_comment",
+                            ),
+                        ]
+                    ),
+                ),
+            ]
+        )
+
+        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+
+    def get_labels(self):
+        classes = {}
+        for issue in self.github.get_issues():
+            # Skip issues with empty title or body
+            if not issue["title"] or not issue["body"]:
+                continue
+
+            # Skip issues that are not moderated yet as they don't have a meaningful title or body
+            if issue["title"] == "In the moderation queue.":
+                continue
+
+            if issue["milestone"] and (
+                issue["milestone"]["title"] == "invalid"
+                or issue["milestone"]["title"] == "incomplete"
+            ):
+                for label in issue["labels"]:
+                    if label["name"] == "wcrt-invalid":
+                        classes[issue["number"]] = 1
+
+            for event in issue["events"]:
+                if event["event"] == "milestoned" and (
+                    event["milestone"]["title"] == "needsdiagnosis"
+                    or event["milestone"]["title"] == "moved"
+                ):
+                    classes[issue["number"]] = 0
```

Comment:
I think if we already classify the report as invalid based on labels, there is no need to check the issue events. Thus, using `elif` meant to prevent doing that. 
```suggestion
            elif any(
                event["event"] == "milestoned"
                and (event["milestone"]["title"] in ("needsdiagnosis", "moved"))
                for event in issue["events"]
            ):
```